en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md

```diff
@@ -0,0 +1,725 @@
+            <td><code>name</code></td>
+            <td>Unique name to identify the connection by.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>base</code></td>
+            <td>The service root URL. The default value is `https://googleads.googleapis.com`.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>clientId</code></td>
+            <td>Client ID of the registered application.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>clientSecret</code></td>
+            <td>Client secret of the registered application.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>refreshToken</code></td>
+            <td>The refresh token that can be used to obtain a new access token.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>developerToken</code></td>
+            <td>The developer token for the Google Ads connection.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>loginCustomerId</code></td>
+            <td>The ID of the manager account. Required when using a manager account to access an customer account.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td><code>tokenEndpoint</code></td>
+            <td>The HTTP endpoint that can be uses to obtain an access token. The default value is `https://www.googleapis.com/oauth2/v3/token`.</td>
+            <td>Yes</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <googleAds.init>
+        <connectionType>googleAds</connectionType>
+        <name>GGOGLE_ADS_CONN</name>
+        <base>https://googleads.googleapis.com</base>
+        <clientId>REPLACE_WITH_CLIENT_ID</clientId>
+        <clientSecret>REPLACE_WITH_CLIENT_SECRET</clientSecret>
+        <refreshToken>REPLACE_WITH_REFRESH_TOKEN</refreshToken>
+        <developerToken>REPLACE_WITH_DEVELOPER_TOKEN</developerToken>
+        <loginCustomerId>REPLACE_WITH_MANAGER_ID</loginCustomerId>
+        <tokenEndpoint>https://www.googleapis.com/oauth2/v3/token</tokenEndpoint>
+    </googleAds.init>
+    ```
+
+## Operations
+
+??? note "createCustomerClient"
+    The `createCustomerClient` operation creates a new client under the provided manager. The new client customer is returned.
+    <table>
+        <tr>
+            <th>Parameter name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td><code>customerId</code></td>
+            <td>The ID of the manager under whom client customer is being created. Type: string</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>customerClient</code></td>
+            <td>The new client customer to create. The resource name on this customer will be ignored. Type: object [Customer](https://developers.google.com/google-ads/api/rest/reference/rest/v17/Customer)</td>
```

Comment:
Shall we update the external links as below:

`For more information, go to [Customer](https://developers.google.com/google-ads/api/rest/reference/rest/v17/Customer).`

There are multiple occurrences.

Reply:
Fixed with [3d8b309](https://github.com/wso2/docs-mi/pull/933/commits/3d8b309f07637aea9f374fc5b18c0cf72322444f)